msmbuilder/feature_selection/base.py

```diff
@@ -0,0 +1,119 @@
+# Author: Carlos Xavier Hernandez <cxh@stanford.edu>
```

Comment:
maybe we should have a timeseries to array concatenator and everything inherits from it. The descriptor can be different but then use the super method to initiate the base class.